app/strings.php

```diff
@@ -0,0 +1,1365 @@
+            __('Is there training on how to design and facilitate and effective consultation?'),
+            __('How will my access needs be met during a consultation?'),
+            __('Does it cost Regulated Organizations to use The Accessibility Exchange services?'),
+            __('Request an introductory phone call'),
+            __('Request a call'),
+            __('Learn about our pricing'),
+            __('Go to pricing'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2336%3A64484
+        'HomePage/participants' => [
+            __('What are the different roles that are played in The Accessibility Exchange'),
+        ],
+    ],
+
+    '1.0 Registration + My dashboard + translations' => [
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1125%3A4220
+        'All/Registration/Password' => [
+            __('I agree with the [terms of service](:url) or using The Accessibility Exchange'),
+            __('I agree with the [privacy policy](:url) or using The Accessibility Exchange'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=3706%3A98663
+        'Dashboard/Ind-CP/Onboarding start' => [
+            __('Role|Roles'),
+            __('Watch introduction video again'),
+            __('Customize this website’s accessibility'),
+            __('Change colour contrast, turn on text to speech, and turn on Sign Language videos.'),
+            __('Customize'),
+            __('Dismiss'),
+            __('Enter your collaboration preferences'),
+            __('Please complete this section so that you can be set up to participate in consultations.'),
+            __('Not started yet'),
+            __('Attend an orientation session'),
+            __('Please attend an orientation session for your roles as :role.|Please attend an orientation session for each of your roles as :firstRole and :lastRole.'),
+            __('Quick links'),
+            __('Projects I’m participating in'),
+            __('Projects I’m participating in as a Consultation Participant.'),
+            __('At present, you have no new notifications.'),
+        ],
+        // https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=1780%3A48437
+        'Dashboard/Ind-CP/Enter personal info-1' => [
+            __('Dashboard'),
+            __('Please enter your collaboration preferences'),
+            __('Matching information'),
+            __('Access needs'),
+            __('Language preferences'),
+            __('Payment information'),
+            __('Communication and consultation preferences'),
+            __('Optional'),
```

Comment:
I've assumed that `(Optional)` will be added separately, similar to the required flag.